src/lib/report-size.ts

```diff
@@ -1,4 +1,4 @@
-import * as colors from 'picocolors'
+import colors from 'picocolors'
```

Comment:
I was getting `colors.* is not a function` errors running in every environment, and while logging it showed that everything was inside the default export.